gce/add-google-cloud-ops-agent-repo.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright 2020 Google Inc. All rights reserved.
+# Copyright 2020,2024 Google, LLC and contributors ; All rights reserved.
```

Comment:
can you update copyright so it's just the first year Google LLC (no comma) and contributors remove the All rights reserved because that means noone can use it. I think it's missing the other lines from the Apache2 License that you can copy from the original file.

Reply:
Sure!  It's already a late night so I'll pick it up tomorrow